### PR TITLE
Fix padding

### PR DESCRIPTION
**Pull request by Sky7hunder**

### CODE REVIEW

**Comment by galulex** [comment]
Where's screenshot?

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -309,12 +319,18 @@ svg#main {
...
 }
 
-.testimonials-title, .services-title {
+.testimonials-title {
   top: 3%;
   right: 50%;
   transform: translateX(50%);
 }
 
+.services-title {
```
Comment (galulex) [inline app/assets/stylesheets/application.scss]: Class duplication

Comment (Sky7hunder) [inline app/assets/stylesheets/application.scss]: Oh, yep. I fixed it.

**Pull request merged**